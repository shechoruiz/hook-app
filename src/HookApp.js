import React from "react";

const HookApp = () => {
  return <h1>HookApp</h1>;
};

export default HookApp;
