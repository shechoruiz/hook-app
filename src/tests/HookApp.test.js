import React from "react";
import "@testing-library/jest-dom";
import { shallow } from "enzyme";
import HookApp from "../HookApp";

describe("Prueba inicial de configuraciÃ³n", () => {
  test("1. Realizar el snapshot del componente", () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
