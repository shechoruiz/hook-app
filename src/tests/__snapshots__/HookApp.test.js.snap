// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba inicial de configuraci√≥n 1. Realizar el snapshot del componente 1`] = `
<h1>
  HookApp
</h1>
`;
